proposals/quick-look-list-pkgs.txt

```diff
@@ -0,0 +1,199 @@
+For extensions, based on LTS 13.6:
+RankNTypes, Rank2Types and ImpredicativeTypes
```

Comment:
Did you intend to commit this file?

Reply:
No, but I thought it would be useful during the discussion.